rfcs/0036-rfc-process-team-amendment.md

```diff
@@ -0,0 +1,202 @@
+   poorly-received. You might want to create a PR in your fork of the RFCs
+   report to help you flesh it out with a few supporters or chat/video
+   conference with a few people involved in the topic of the RFC.
+3. Submit a pull request. As a pull request the RFC will receive design feedback
+   from the larger community, and the author should be prepared to revise it in
+   response.
+4. For the nomination process for potential members of the RFC Shepherd Team,
+   that is specific to each RFC, anyone interested can either nominate another
+   person or themselves to be a potential member of the RFC Shepherd Team. This
+   can already be done when submitting the PR.
+5. The RFC Steering Committee assigns a subset of the nominees to the RFC
+   Shepherd Team and designates a leader for it. This has to be done
+   unanimously.
+6. Build consensus and integrate feedback. RFCs that have broad support are much
+   more likely to make progress than those that don't receive any comments. Feel
+   free to reach out to the RFC Shepherd Team leader in particular to get help
+   identifying stakeholders and obstacles. We would like to encourage reviewers
+   to only make comments on the content of the RFC and reach out to the author
+   directly (via IRC, e-mail, etc.) for wording or typos.
+7. The RFC Shepherd Team will discuss the RFC pull request, as much as possible
+   in the comment thread of the pull request itself. Discussion outside of the
+   pull request, either offline or in a video conference, that might be
+   preferable to get to a solution for complex issues, will be summarized on the
+   pull request comment thread.
+8. RFCs rarely go through this process unchanged, especially as alternatives and
+   drawbacks are shown. You can make edits, big and small, to the RFC to clarify
+   or change the design, but make changes as new commits to the pull request,
+   and leave a comment on the pull request explaining your changes.
+   Specifically, do not squash or rebase commits after they are visible on the
+   pull request.
+9. At some point, a member of the RFC Shepherd Team will propose a "motion for
+final comment period" (FCP), along with a disposition for the RFC (merge, close,
+or postpone).
+    * This step is taken when enough of the tradeoffs have been discussed that
+      the RFC Shepherd Team is in a position to make a decision. That does not
+      require consensus amongst all participants in the RFC thread (which is
+      usually impossible). However, the argument supporting the disposition on
+      the RFC needs to have already been clearly articulated, and there should
+      not be a strong consensus against that position outside of the RFC
+      Shepherd Team. RFC Shepherd Team members use their best judgment in taking
+      this step, and the FCP itself ensures there is ample time and notification
+      for stakeholders to push back if it is made prematurely.
```

Comment:
Might be good to reference https://tools.ietf.org/html/rfc7282 as a guideline, or even a formal expectation.

Reply:
Yes, it won't hurt to at least link what we mean by "consensus", and I believe this particular link is a good one.  I expect longer-term participants from a similar community to mostly have a very similar notion of what it means, but still...

Reply:
Fine leaving this for now, but would be worth returning to if/when we hit more contentious RFCs